src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -31,9 +32,11 @@
 
 #include <lib/support/CodeUtils.h>
 #include <lib/support/ReturnMacros.h>
+#include <lib/support/SafeInt.h>
 #include <lib/support/logging/CHIPLogging.h>
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/ConnectivityManager.h>
+#include <platform/ThreadStackManager.h>
```

Comment:
This breaks the Mac build because there is no `platform/Darwin/ThreadStackManagerImpl.h` header....

Reply:
It seems like the thread bits should all be conditioned on CHIP_ENABLE_OPENTHREAD, which they are not?

Reply:
Done